src/model.js

```diff
@@ -318,7 +333,8 @@ export default class Model extends Object3D {
         Instanced: attribute.instanced,
         Verts: attribute.value.length / attribute.size,
         Size: attribute.size,
-        Bytes: attribute.value.length * attribute.value.BYTES_PER_ELEMENT
+        Bytes: attribute.value.length * attribute.value.BYTES_PER_ELEMENT,
+        location: program && program.attributeLocations[attributeName]
```

Comment:
Use capital (Location) for consistency with other table titles?

Also, if location is undefined, it would be cool to check the bufferType and if it is ELEMENTS_ARRAY_BUFFER print something to indicate that this is the indices, perhaps simply print "ELEMENTS_ARRAY_BUFFER"?

If so, we should probably move the Location string calculation logic up above the object.